client/src/document/index.scss

```diff
@@ -3,15 +3,29 @@
 @import "~@mdn/minimalist/sass/vars/borders";
 @import "~@mdn/minimalist/sass/vars/typography";
 
+@import "~@mdn/minimalist/sass/mixins/animation";
 @import "~@mdn/minimalist/sass/mixins/utils";
 
 @import "./prism-overrides";
 
-.loading-document-placeholder article {
-  // Any huge margin bottom to keep the footer away when loading
-  margin-bottom: 1200px;
+/* we already disable all animation and transitions globally via a reset so */
+.page-content-container {
+  /* stylelint-disable a11y/media-prefers-reduced-motion */
+  &.loading-document-placeholder {
+    @include slow-pulse();
+    min-height: 55vh;
+    width: 100%;
+
+    @media #{$mq-small-desktop-and-up} {
+      margin: $base-spacing auto;
+      max-width: $max-width-default;
+      min-height: 75vh;
+      width: 50%;
+    }
+  }
 }
 
+/* stylelint-enable */
```

Comment:
Shouldn't this comment sit immediately after the `}` bracket for the `&.loading-document-placeholder {...}`  block?

Reply:
Turns out, as I am adding the animation via a mixin now, the code has linter has no idea I am even using `animation` and so those comments are no longer needed :+1: